data/mods/Magiclysm/itemgroups/itemgroups.json

```diff
@@ -101,9 +101,12 @@
     "id": "magic_shop_books",
     "items": [
       { "group": "spellbook_tier_0", "prob": 60 },
-      { "group": "spellbook_tier_1", "prob": 50 },
-      { "group": "spellbook_tier_2", "prob": 35 },
-      { "group": "spellbook_tier_3", "prob": 20 },
+      { "group": "spellbook_tier_1", "prob": 40 },
```

Comment:
these should be probably combined to use a `spellbook_loot_` tier (found in magiclysm/itemgroups/spellbooks.json)